runtime/doc/vim9class.txt

```diff
@@ -690,8 +690,8 @@ value for the object variables will be used.  This is a more useful example,
 with default values: >
 
 	class TextPosition
-	   this.lnum: number = 1
-	   this.col: number = 1
+	   var lnum: number = 1
+	   var col: number = 1
 	endclass
 
 If you want the constructor to have mandatory arguments, you need to write it
```

Comment:
The "Declaring object and class variables" section needs to be updated to reflect the rationale for using the "var" prefix now.